test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -1854,6 +1854,37 @@ public string F2(int a)
             }
         }
 
+        [Fact]
+        public void PackCommand_VersionSuffixIsAssigned()
+        {
+            var nugetexe = Util.GetNuGetExePath();
+            var workingDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
```

Comment:
Don't use path.GetTempPath, see the approach in the follwing PR - https://github.com/NuGet/NuGet.Client/pull/122/files